src/filter.rs

```diff
@@ -41,15 +42,27 @@ pub fn get_biggest(
 fn add_children<'a>(
     using_a_filter: bool,
     min_size: Option<usize>,
+    only_dir: bool,
     file_or_folder: &'a Node,
     depth: usize,
     mut heap: BinaryHeap<&'a Node>,
 ) -> BinaryHeap<&'a Node> {
     if depth > file_or_folder.depth {
-        heap.extend(file_or_folder.children.iter().filter(|c| match min_size {
-            Some(ms) => c.size > ms as u64,
-            None => !using_a_filter || c.name.is_file() || c.size > 0,
-        }))
+        // heap.extend(file_or_folder.children.iter().filter(|c| c.name.is_dir()))
```

Comment:
delete this line and run cargo format on this file and we should be good to go :)

Reply:
I cleared code and cargo fmt.

I did rebase -i ... to clean up commit message.